app/presenters/publishing_api/case_study_presenter.rb

```diff
@@ -66,7 +66,7 @@ def body
 
     def image_details
       {
-        url: Whitehall.public_asset_host + presented_case_study.lead_image_path,
+        url: presented_case_study.lead_image_path,
```

Comment:
In this commit note (for "Use correct image URL in CaseStudyPresenter") you've said, "I've added one [test] so that we prevent this issue again.", but I can't see any new tests. Did you intend to add a test or decide that it wasn't worth it in the end?

Reply:
I think I meant "I've changed the tests to prevent this issue again". I'll fix the commit note.